BothamNetworkingTests/ConnectionErrorHTTPClient.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  ConnectionErrorHTTPClient.swift
+//  BothamNetworking
+//
+//  Created by Pedro Vicente Gomez on 06/01/16.
+//  Copyright © 2016 GoKarumi S.L. All rights reserved.
+//
+
+import Foundation
+import Result
+@testable import BothamNetworking
+
+class ConnectionErrorHTTPClient: HTTPClient {
+    func send(httpRequest: HTTPRequest, completion: (Result<HTTPResponse, NSError>) -> ()) {
+        let connectionError = NSError(domain: "NSURLErrorDomain", code: -1009, userInfo: nil)
```

Comment:
There is a constant for NSURLErrorDomain string.
https://developer.apple.com/library/mac/documentation/Cocoa/Reference/Foundation/Miscellaneous/Foundation_Constants/index.html#//apple_ref/doc/constant_group/NSURL_Domain